recipes/cost733class/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "cost733class" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lochbika/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dd883ac706c30da80184409b69d2196c1fa1da0073af62dffab434c1373d91a5
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+
+    # Unix
```

Comment:
With the pre-processor selectors this comment and the one below aren't really needed, right?

```suggestion
```